pages/About Data Triad.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+    <div class="container">
+        <article>
+            <h1 class:"title">About Data Triad</h1>
+    <div class="container">
+        <article>
+            <h2 class:"title">Vision</h2>
+            <p> Vision: Cultivating a dynamic community of local and virtual data scientists focused on employing data for actionable and engaging applications.</p>
+    <div class="container">
+        <article>        
+            <h2 class:"title">Mission</h2>
+            <p> Empowering and enhancing the practical application of data science.</p>
+    <div class="container">
+        <article>
+            <h2 class:"title">Values</h2>
+            <p> Community, Development, Application of Ideas</p>
+    <div class="container">
+        <article>
+            <h2 class:"title">Areas of Recent Focus</h2>
+            <p> Areas of recent focus: Machine Learning, Artificial Intelligence, Data Science, Data Analytics, Data Engineering, Big Data, Python, R, TensorFlow, Deep Learning, GANs to name a few.</p>
+<template>
+
+
+  
+<template>
+  <div v-if="$fetchState.pending">
+    <p> Events are loading... </p>
+  </div>
+  <div v-else class="eventList">
+    <ul>
+      <li v-for="event in upcomingEvents" :key="event.id">
+        <a :href="event.link">
+          {{ formatDate(event.local_date) }}, {{ formatTime(event.local_time) }}
+        </a>
+        <br>
+        <strong>{{ event.name }}</strong>
+      </li>
+    </ul>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'nuxt-property-decorator';
+
+@Component
+export default class Events extends Vue {
```

Comment:
The class name will be the name of the file. Currently there are spaces in the file name `About Data Triad.vue` which isn't allowed. Could be `about.vue`, or `about-data-triad.vue`. If you go with `about.vue` then the class will be About. It looks like you are also trying to use the events component here so your script will be like this.

```ts
import { Component, Vue } from 'nuxt-property-decorator';
import Events from '@/components/Events.vue';

@Component({
  components: {
    Events,
  },
})
export default class About extends Vue {}
```